mmv1/products/compute/RegionUrlMap.yaml

```diff
@@ -244,6 +244,7 @@ properties:
           # exactly_one_of:
           #   - path_matchers.0.default_service
           #   - path_matchers.0.default_url_redirect
+          required: true
```

Comment:
Instead of making this field `required`, we can add a note to the `description` of this field that one of `default_service` and `default_url_redirect` should be provided.